authorization/token/worker/token_cleanup_worker.go

```diff
@@ -0,0 +1,58 @@
+package worker
+
+import (
+	"context"
+	"github.com/fabric8-services/fabric8-auth/application"
+	"github.com/fabric8-services/fabric8-auth/log"
+	"time"
+)
+
+type TokenCleanupWorker interface {
+	Start(ticker *time.Ticker)
+	Stop()
+}
+
+func NewTokenCleanupWorker(ctx context.Context, app application.Application) TokenCleanupWorker {
+	return &tokenCleanupWorker{
+		ctx: ctx,
+		app: app,
+	}
+}
+
+type tokenCleanupWorker struct {
+	ctx    context.Context
```

Comment:
You don't use ctx which makes sense. So, let's remove it.

Reply:
It's used further down in the cleanup thread, where it's passed to the token service.